coralproject/talk
# CPSRE-1015: migrate coral off of CircleCI and move to GHA

## What does this PR do?

This PR moves the `build-test` and `build-test-deploy` steps off CircleCI and onto Github Actions.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?



## Does this PR introduce any new environment variables or feature flags? 



## If any indexes were added, were they added to `INDEXES.md`?



## How do I test this PR?
Make sure that `build-test` is working, and I have already tried to mimic the `build-test-deploy` step by simulating using a tag `v9.9.9`
 
 
## How do we deploy this PR?
